projects/assets/src/translations/en/common.ts

```diff
@@ -20,8 +20,8 @@ export const common = {
       title_plural: '{{count}} results for {{query}}',
     },
     checkout: {
-      title: 'Checkout {{count}} item',
-      title_plural: 'Checkout {{count}} items',
+      title: 'Checkout',
+      title_plural: 'Checkout',
```

Comment:
I this case I think we can remove `title_plural` key. Also I would mention these changes in the migration doc